content/MBC2.md

```diff
@@ -18,7 +18,7 @@ battery to save data during power-off though. As the data consists of
 4bit values, only the lower 4 bits of the "bytes" in this memory area
```

Comment:
4-bit, also why the quotes around "bytes"?

Reply:
Fixed